docs/content/preview/api/ysql/syntax_resources/ysql_grammar.ebnf

```diff
@@ -531,6 +531,8 @@ create_index = 'CREATE' [ 'UNIQUE' ] 'INDEX' [ 'CONCURRENTLY' | 'NONCONCURRENTLY
                '(' index_elem { ',' index_elem } ')' \
                [ 'INCLUDE' '(' column_name { ',' column_name } ')' ] \
                [ 'TABLESPACE' tablespace_name ] \
+               [ 'SPLIT' ( 'INTO' integer 'TABLETS'
+                      | 'AT' 'VALUES' '(' split_row { ',' split_row } ')' ) ]
```

Comment:
Could indentation be better?

Reply:
I just copied what was used on `CREATE TABLE` grammar. 

Can you suggest a better one?